src/impact-dashboard/EpidemicModelContext.tsx

```diff
@@ -52,12 +81,65 @@ function epidemicModelReducer(
 
 function EpidemicModelProvider({ children }: EpidemicModelProviderProps) {
   const [state, dispatch] = React.useReducer(epidemicModelReducer, {
-    stateCode: "US",
-    totalIncarcerated: populationAndHospitalData.US.incarceratedPopulation,
+    stateCode: "US Total",
+    countyName: "Total",
+    countyLevelDataLoading: true,
     rateOfSpreadFactor: 3.7,
     usePopulationSubsets: false,
   });
 
+  // fetch from external datasource
+  React.useEffect(() => {
+    fetch(
+      "https://docs.google.com/spreadsheets/d/e/2PACX-1vSeEO7JySaN21_Cxa7ON_x" +
+        "UHDM-EEOFSMIjOAoLf6YOXBurMRXZYPFi7x_aOe-0awqDcL4KZTK1NhVI/pub?gid=" +
+        "1836987932&single=true&output=csv",
+    ).then(async (response) => {
```

Comment:
nit: if you're going to use `.catch`, you can just do:

```tsx
let response = await fetch(...)
// all the code below can just go here
```

See here for an example of how I did it for news feed events:
https://github.com/Recidiviz/covid19-dashboard/blob/master/src/news-feed-events/NewsFeedEvent.tsx#L24-L37

Reply:
I couldn't do that because you apparently can't pass an async function to `useEffect` ... the alternative would be to move all this to an inner function and then immediately call the inner function which didn't seem obviously better than just doing this

Reply:
Oh I see what you mean, sorry I forgot about that. The idiom that I've learned to learned to use is:

```tsx
useEffect(() => {
  async function effect(){
    ...
  }
  effect()
})

```

Obviously, has some of the same drawbacks of this solution, but at least we're not special-casing the first line to have to be written some other kind of way.

Reply:
yeah i guess that makes sense, i'll change it